terra-workspace-manager-client/build.gradle

```diff
@@ -39,31 +39,16 @@ repositories {
 
 dependencies {
     ext {
-        commonsLang3 = "3.9"
-        googleApiServicesOauth2 = "v2-rev20190313-1.30.1"
-        googleClient = "1.30.7"
-        googleCloudStorage = "1.103.1"
-        googleOauthClientJetty = "1.30.5"
-        jackson = "2.11.0"
-        jersey = "2.30.1"
-        junit = "4.13"
-        swagger = "1.6.0"
-        hamcrest = "2.2"
+        jersey = "2.31"
     }
 
-    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: "${jackson}"
-    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: "${jackson}"
     implementation group: 'org.openapitools', name: 'jackson-databind-nullable', version: '0.2.1'
-    implementation group: 'io.swagger', name: 'swagger-annotations', version: "${swagger}"
+    implementation group: 'io.swagger', name: 'swagger-annotations', version: '1.6.1'
     implementation group: 'org.glassfish.jersey.core', name: 'jersey-client', version: "${jersey}"
     implementation group: 'com.google.code.findbugs', name: 'jsr305', version: '3.0.2'
     implementation group: 'org.glassfish.jersey.media', name: 'jersey-media-json-jackson', version: "${jersey}"
     implementation group: 'org.glassfish.jersey.media', name: 'jersey-media-multipart', version: "${jersey}"
-    implementation "org.glassfish.jersey.inject:jersey-hk2:${jersey}"
-    implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: "${jackson}"
-
-    testImplementation group: 'junit', name: 'junit', version: '4.13'
-    testImplementation group: 'org.hamcrest', name: 'hamcrest', version: '2.2'
```

Comment:
Do we not need all these dependencies?

If not, how did you find that out? I'd love to do better dependency management in the future

Reply:
So, there are only two tasks that happen through this sub-project, the openApi build/jar and the actual publishing.

There are no tests here, so that was an easy start.
As for the rest, I commented out things and ran the build steps. I ran `:terra-workspace-manager-client:artifactoryPublish --dry-run` to figure out what needed to pass without doing an actual publish, which is mainly `jar`.